### ISSUE TITLE
Incorrect German sentence in dialog window 

### ISSUE THREAD

**Issue by Komstel** [dmitryd/dd_deepl#40]
When using the translation tool, an incorrect or incomprehensible sentence is displayed in the dialog box in connection with the DeepL plug-in. It says: "Die Übersetzungen werden automatisch mit DeepL erstellt. Die übersetzten Elemente werden mit den Elementen in der Originalsprache verbunden. Stellen Sie sicher, dass Sie keine oder sensible Informationen aus Ihrem Unternehmen heraus." 
The last sentence is incorrect, please add a verb or delete the sentence completely. 
![screenshot-deepl-plugin](https://github.com/user-attachments/assets/591924f1-b11f-4fcd-b33b-24c2e0d1a216)

**Comment by dmitryd** [comment]
I do not speak German. The translation was provided by DeepL.

**Comment by Komstel** [comment]
I understand that, but would you please be so kind as to delete the sentence or change it as follows: 
"Stellen Sie sicher, dass Sie keine sensiblen Informationen aus Ihrem Unternehmen weitergeben." (means: Make sure you don't share any sensitive information from your company.)
Otherwise I will always get inquiries from editors as to what this sentence means.

**Comment by dmitryd** [comment]
Well, it is better that they are warned and that you explain it to them. You are sending data to a 3rd party company! You need a warning like this there!

**Comment by Komstel** [comment]
I agree, but the sentence in the original DeepL version is not a warning, it's just incomplete/incomprehensible.
